core/src/datayoga_core/step_buffer.py

```diff
@@ -6,8 +6,8 @@
 logger = logging.getLogger("dy")
 
 class StepBuffer(Step):
-    def __init__(self, id, min_buffer_size=4, max_buffer_size=4, flush_ms=1000):
-        super().__init__(id, None)
+    def __init__(self, step_id, min_buffer_size=4, max_buffer_size=4, flush_ms=1000):
+        super().__init__(step_id, None)
```

Comment:
If None is supported for block, this should be changed to in step:
```
    def __init__(self, step_id: str, block: Optional[Block], concurrency=1):
```

Reply:
✅